src/manifestoo/main.py

```diff
@@ -253,6 +282,35 @@ def list_depends(
     )
 
 
+@app.command()
+def list_codepends(
+    ctx: typer.Context,
+    separator: Optional[str] = typer.Option(
+        None,
+        help="Separator character to use (by default, print one item per line).",
+    ),
+    transitive: bool = typer.Option(
+        True,
+        "--transitive",
+        help="Print all transitive co-dependencies.",
+    ),
+    include_selected: bool = typer.Option(
+        True,
+        "--include-selected",
+        help="Print the selected addons along with their co-dependencies.",
```

Comment:
Could you explain here what co-dependencies are ?

Reply:
Given a relation, it's dual (co-relation) is obtained by inverting the arrows. But I will put an explanation.